integration/mediation-tests/tests-transport/src/test/java/org/wso2/carbon/esb/jms/securevault/test/JMSTransportSecureVaultTest.java

```diff
@@ -0,0 +1,186 @@
+
+    private static final String ACTIVEMQ_BROKER = "activemq-broker-5.9.1.jar";
+    private static final String ACTIVEMQ_CLIENT = "activemq-client-5.9.1.jar";
+    private static final String GERONIMO_J2EE_MANAGEMENT = "geronimo-j2ee-management_1.1_spec-1.0.1.jar";
+    private static final String GERONIMO_JMS = "geronimo-jms_1.1_spec-1.1.1.jar";
+    private static final String HAWTBUF = "hawtbuf-1.9.jar";
+    private static final String ACTIVEMQ_KAHADB_STORE = "activemq-kahadb-store-5.9.1.jar";
+    private static final String SLF4J_API = "slf4j-api-1.7.13.jar";
+    private static final String ACTIVEIO_CORE = "activeio-core-3.1.4.jar";
+
+    /**
+     * The embedded broker instance used for the test.
+     */
+    private JMSBroker activeMQBroker;
+    /**
+     * The simple authentication plugin for the broker to require logging in.
+     */
+    private SimpleAuthenticationPlugin simpleAuthenticationPlugin;
+
+    @BeforeClass(alwaysRun = true)
+    public void init() throws Exception {
+        super.init();
+        serverConfigurationManager = new ServerConfigurationManager(context);
+        String secureVaultConfDir = FrameworkPathUtil.getSystemResourceLocation() + "artifacts" + File.separator
+                + "ESB" + File.separator  + "jms" + File.separator + "securevault" + File.separator;
+        String carbonSecurityDir = CarbonBaseUtils.getCarbonHome() + File.separator + "conf" + File.separator
+                + "security" + File.separator;
+
+        String srcFileLocation = secureVaultConfDir + CIPHER_TOOL_PROPERTIES_FILE;
+        String targetFileLocation = carbonSecurityDir + CIPHER_TOOL_PROPERTIES_FILE;
+        File srcFile = new File(srcFileLocation);
+        File targetFile = new File(targetFileLocation);
+        serverConfigurationManager.applyConfigurationWithoutRestart(srcFile, targetFile, true);
+
+        srcFileLocation = secureVaultConfDir + CIPHER_TEXT_PROPERTIES_FILE;
+        targetFileLocation = carbonSecurityDir + CIPHER_TEXT_PROPERTIES_FILE;
+        srcFile = new File(srcFileLocation);
+        targetFile = new File(targetFileLocation);
+        serverConfigurationManager.applyConfigurationWithoutRestart(srcFile, targetFile, true);
+
+        srcFileLocation = secureVaultConfDir + SECRET_CONF_PROPERTIES_FILE;
+        targetFileLocation = carbonSecurityDir + SECRET_CONF_PROPERTIES_FILE;
+        srcFile = new File(srcFileLocation);
+        targetFile = new File(targetFileLocation);
+        serverConfigurationManager.applyConfigurationWithoutRestart(srcFile, targetFile, true);
+
+        srcFileLocation = secureVaultConfDir + AXIS2_XML_FILE;
+        targetFileLocation = CarbonBaseUtils.getCarbonHome() + File.separator + "conf" + File.separator + "axis2"
+                + File.separator + AXIS2_XML_FILE;
+        srcFile = new File(srcFileLocation);
+        targetFile = new File(targetFileLocation);
+        serverConfigurationManager.applyConfigurationWithoutRestart(srcFile, targetFile, true);
```

Comment:
does apply without restarting will apply these configs ?

Reply:
A [restart is done](https://github.com/MaryamZi/product-ei/blob/f6f78695fddb2071a30875f96f987b4cc2a555b4/integration/mediation-tests/tests-transport/src/test/java/org/wso2/carbon/esb/jms/securevault/test/JMSTransportSecureVaultTest.java#L131) after all configurations are applied and relevant JARs are copied.